ui/components/component-library/avatar-token/avatar-token.js

```diff
@@ -3,19 +3,18 @@ import classnames from 'classnames';
 import PropTypes from 'prop-types';
 import Box from '../../ui/box/box';
 import { AvatarBase } from '../avatar-base';
-
 import {
   COLORS,
-  SIZES,
   DISPLAY,
   ALIGN_ITEMS,
   JUSTIFY_CONTENT,
 } from '../../../helpers/constants/design-system';
+import { AVATAR_TOKEN_SIZES } from './avatar-token.constants';
 
 export const AvatarToken = ({
-  size = SIZES.MD,
-  tokenName,
-  tokenImageUrl,
+  size = AVATAR_TOKEN_SIZES.MD,
```

Comment:
I think we've come up with a convention to use component specific const that wrap general design system conts for argTypes and propTypes only. Because `AVATAR_TOKEN_SIZES` is made up of `SIZES` use it for argTypes `options: Object.values(AVATAR_TOKEN_SIZES),` and propTypes`PropTypes.oneOf(Object.values(AVATAR_TOKEN_SIZES))` only. Use `SIZES` for everything else, docs, default props etc. We are suggesting the use of `SIZES` in general but limiting the options using `AVATAR_TOKEN_SIZES` 

```suggestion
  size = SIZES.MD,
```